fastlane/lanes/localization.rb

```diff
@@ -196,9 +194,8 @@
         play_store_desc: { desc: 'full_description.txt', max_size: 4000 }
       }
       unless skip_release_notes
-        delete_old_changelogs(app: app, build: build_number)
         version_suffix = version.split('.').join
-        files["release_note_#{version_suffix}"] = { desc: "changelogs/#{build_number}.txt", max_size: 500, alternate_key: "release_note_short_#{version_suffix}" }
+        files["release_note_#{version_suffix}"] = { desc: 'changelogs/default.txt', max_size: 500, alternate_key: "release_note_short_#{version_suffix}" }
```

Comment:
What happens if the release notes for a locale is skipped by the `gp_downloadmetadata` action though? (e.g. because its translation exceeds the `max_size` and so does the translation of its `releae_note_short` so `gp_downloadmetadata` ends up skipping it; or because the translation for the new release notes of the latest version has not yet been submitted or approved in GlotPress so is not part of the export when queried by the API, …)

I think in those cases there's a risk that the content of the corresponding `fastlane/metadata/android/{locale}/changelogs/default.txt` file will stay untouched, and will thus retain its previous content from the previous release—as opposed to do what we want which would be to be set to an empty file, for _fastlane_ to detect and for Google Play to fallback to displaying the English release notes for the new version in the offending locale(s), instead of displaying the translated release notes of the previous version…